packages/blender.rb

```diff
@@ -1,45 +1,66 @@
   binary_compression 'tar.zst'
 
   binary_sha256({
-     x86_64: '2ef554553a2f40d3fc4a68de2389502b087a0f79c0421acb2e3445dea3224f50'
+     x86_64: '1785f34e092a4c359f003b6bf70ae1a39c8b06eb9f4a89aa9b729e1893bfc4e4'
   })
 
-  depends_on 'sommelier' # L
-  depends_on 'gcc_lib' # R
-  depends_on 'glibc' # R
-  depends_on 'libglvnd' # R
-  depends_on 'libice' # R
-  depends_on 'libsm' # R
+  # depends_on 'audaspace'
+  depends_on 'boost' # R
+  depends_on 'eigen'
+  depends_on 'embree'
+  depends_on 'ffmpeg'
+  depends_on 'freetype'
+  depends_on 'gmp' # R
+  depends_on 'jack' # R
+  depends_on 'libepoxy'
+  depends_on 'libjpeg_turbo'
+  depends_on 'libpng'
+  depends_on 'libsdl2' # R
+  depends_on 'libsndfile' # R
+  depends_on 'libtiff'
+  depends_on 'libwebp' # R
   depends_on 'libx11' # R
-  depends_on 'libxext' # R
   depends_on 'libxfixes' # R
+  depends_on 'libxi' # R
   depends_on 'libxkbcommon' # R
   depends_on 'libxrender' # R
-  depends_on 'libxt' # R
   depends_on 'libxxf86vm' # R
-  depends_on 'mesa' # R
-  depends_on 'ncurses' # R
-  depends_on 'zlibpkg' # R
-  depends_on 'zstd' # R
+  depends_on 'openal' # R
+  depends_on 'openexr' # R
+  depends_on 'openimageio'
+  depends_on 'openjpeg' # R
+  depends_on 'pipewire' # R
+  depends_on 'pugixml' # R
+  depends_on 'pulseaudio' # R
+  depends_on 'py3_numpy'
+  depends_on 'py3_requests'
+  depends_on 'python3'
+  depends_on 'tbb' # R
+  depends_on 'zlibpkg'
+  depends_on 'zstd'
 
-  def self.install
-    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/bin"
-    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/share/blender"
-    FileUtils.mv Dir['*'], "#{CREW_DEST_PREFIX}/share/blender"
-    FileUtils.ln_s "#{CREW_PREFIX}/share/blender/blender", "#{CREW_DEST_PREFIX}/bin/blender"
+  def self.patch
+    # These fixes haven't made their way to a release yet, so we just patch them in.
+    # [PATCH] Use set_and_warn_dependency to disable WITH_MOD_OCEANSIM if WITH_FFTW3 is disabled.
+    downloader 'https://projects.blender.org/blender/blender/commit/0a74065a1995ef5423b3e073f53328d664b43b05.patch', '8a2f9739c716822a94c94894cf32b4ed8d87809b38a8c53d6288f5f2ce8490f1'
+    system 'patch -Np1 -i 0a74065a1995ef5423b3e073f53328d664b43b05.patch'
+    # [PATCH] Unbreak lite-build WITH_GMP enabled
+    downloader 'https://projects.blender.org/blender/blender/commit/861536b24c866d09857ea4b92717f822da4bc75e.patch', 'c7b6560da9062aa984aba3b67212d4694cbd3ec4a72dc6cae3b3d01a43ed0b5a'
+    system 'patch -Np1 -i 861536b24c866d09857ea4b92717f822da4bc75e.patch'
   end
 
-  def self.postinstall
-    ExitMessage.add "Type 'blender' to get started.".lightblue
-  end
+  # Blender needs the single-precision build of fftw, which we do not provide.
+  # WITH_SYSTEM_AUDASPACE is broken in 4.1.1, should be resolved in 4.2
+  # WITH_INSTALL_PORTABLE needs to be disabled to get a a FHS-compatible install.
+  cmake_options '-DWITH_FFTW3=OFF -DWITH_SYSTEM_EIGEN3=ON -DPYTHON_VERSION=3.12 -DWITH_PYTHON_INSTALL=OFF -DWITH_INSTALL_PORTABLE=OFF'
```

Comment:
I just noticed we might want to do the `#{version.split('y').last}` trick here too.  That would mean `version '4.1.1-py3.12'` for this package.

Reply:
Blender doesn't always require a hard-defined python version, as it does officially support python 3.11-- Once it upgrades to 3.12, there is no need for this define.

So, my thinking is that it makes sense for this version to be manually updated, so it can be removed if unneeded.

Reply:
I guess that reasoning makes sense but we'll just need to remember to bump the version on the next release, if available.  Not a simple thing to remember.